app/controllers/admin/cms/collections_controller.rb

```diff
@@ -46,7 +46,7 @@ def resource_params
       :call_number,
       :description,
       :url,
-      :image_url,
+      :image,
       :vendor_id
```

Comment:
New images that are uploaded for a collection will be stored in a new image column on the table.